backend/app/Models/Traits/ReadStatus.php

```diff
@@ -0,0 +1,62 @@
+<?php
```

Comment:
Really like the use of a trait here.

Reply:
You originally developed that so I should hope you like it!